monai/handlers/mlflow_handler.py

```diff
@@ -167,17 +169,52 @@ def start(self, engine: Engine) -> None:
         Check MLFlow status and start if not active.
 
         """
-        mlflow.set_experiment(self.experiment_name)
-        if mlflow.active_run() is None:
+        self._set_experiment()
+        if not self.experiment:
+            raise ValueError(f"Failed to experiment '{self.experiment_name}' as the active experiment")
```

Comment:
Hi @SachidanandAlle ,
      Maybe the error print should be `Failed to set experiment ....`.  A verb like `set` or `start` should be added here. 

Thanks,
Bin